src/QMCWaveFunctions/WaveFunctionComponent.h

```diff
@@ -170,6 +170,11 @@ class WaveFunctionComponent : public QMCTraits
   /** print the state, e.g., optimizables */
   virtual void reportStatus(std::ostream& os) = 0;
 
+  /** Register the component with the TWFPrototype wrapper.  Pure virtual, so will throw
+   * an error at compile time
```

Comment:
Comment is incorrect. This function is not pure virtual, and will throw an error at runtime.

Reply:
Good catch.  I experimented with something that ended up not working.

Reply:
Done.